packages/meditrak-app/app/entityMenu/EntityList.js

```diff
@@ -17,27 +17,36 @@ import {
   getGreyShade,
 } from '../globalStyles';
 import { EntityItem, ITEM_HEIGHT } from './EntityItem';
+import { fetchEntities } from './helpers';
 
 const SEARCH_BOX_HEIGHT = 40;
+const ENOUGH_SEARCH_RESULTS = 20;
```

Comment:
```suggestion
const ENOUGH_SEARCH_RESULTS = 100;
```
It's mentioned a couple times in the ticket that we want 100 results to be the point where we add the 'mid string ordering' results?

Reply:
Yes sorry I was just leaving it until a chat with Flic. Had that now and she went with 500 (was using 100 as another arbitrary example in the Linear comment - apologies for the confusion)